internal/app/options/options.go

```diff
@@ -33,6 +34,17 @@ type Options struct {
 	Pprof bool
 
 	LoadLevel string
+	InitialSyncNumParallelCopiers int
```

Comment:
It seems all of them are only used for Cosmos right now? Then their names should start with Cosmos